jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -223,6 +225,15 @@ public Span log(long timestampMicroseconds, Map<String, ?> fields) {
         if (logs == null) {
           this.logs = new ArrayList<LogData>();
         }
+
+        if (tracer.expandExceptionLogs()) {
+          Map<String, Object> errorFields = exceptionLogs(fields);
+          if (!errorFields.isEmpty()) {
+            errorFields.putAll(fields);
+            fields = errorFields;
```

Comment:
ohhh, putAll puts fields into errorFields and not the other way round (duhhh).

Given that, this means if the user specifically added a "error.object", "stack", "message", and "error.kind", that will overwrite the k,v in errorFields? sounds right.